app/upgrades/v2/upgrades.go

```diff
@@ -19,10 +20,11 @@ func CreateUpgradeHandler(
 		// - https://docs.cosmos.network/master/building-modules/upgrade.html#registering-migrations
 		// - https://docs.cosmos.network/master/migrations/chain-upgrade-guide-044.html#chain-upgrade
 
-		// migrate claims and ERC20 module, other modules are left as-is to
+		// migrate explicitly specified modules, other modules are left as-is to
 		// avoid running InitGenesis.
 		vm[claimstypes.ModuleName] = 1
 		vm[erc20types.ModuleName] = 1
+		vm[inflationtypes.ModuleName] = 1
```

Comment:
nice, we should definitely test this with an upgrade/migration test!

Reply:
btw, I don't think this migration can happen in this handler... this is for v2. The chain will start with v2 and not run this handler

Reply:
@fedekunze doesn't this have to happen in v3, or at least a different upgrade handler in general?